### DISCUSSION TITLE
About energy models and efficient frequencies

### DISCUSSION THREAD

**Discussion by ArbitraryFox** [kdrag0n/freqbench#61] [Q&A] [answered]
Aight, say I have already made an energy model and added it in. Then, I calculated my efficient frequencies and want to use them. Do I recalculate the energy model again but only with the chosen frequencies? Or stick to the original model made?

**Comment by kdrag0n** [comment, marked as answer]
Don't remember exactly what I used to do, but don't see any reason you'd have to recalculate it. Removing frequencies from the model should be enough.

dynamic-power-coefficient might be slightly more accurate if you recalculate it though.
